var/spack/repos/builtin/packages/r-ebseq/package.py

```diff
@@ -6,7 +6,7 @@
 from spack.package import *
 
 
-class BioconductorEbseq(RPackage):
+class REbseq(RPackage):
```

Comment:
If i read the diff correctly, this one was not restored and deprecated.

Reply:
Sorry!, forgot that one :facepalm:

Reply:
No worries, thanks!